modal.js

```diff
@@ -87,7 +87,7 @@ data.forEach((elem) => {
   cardHolder.insertAdjacentHTML('beforeend', myHtmlArticle(elem));
 });
 
-// Popup section
+// Popup section//
 function openPopup() {
   document.querySelector('#popup-section').innerHTML += `<div class="popup-container">
     <div class="popup" id="popup"> 
```

Comment:
- [ ] it would be better if you make the pop up window buttons the same as the one in the Figma and try to remove the underline Also please make sure to implement button effect on hover and pressed as in the Figma design 
- Yours 
<img width="374" alt="Screenshot 2022-11-21 at 15 19 54" src="https://user-images.githubusercontent.com/61790587/203078743-6290313f-5bc1-475a-9238-931936249d37.png">
- Figma
<img width="431" alt="Screenshot 2022-11-21 at 15 24 02" src="https://user-images.githubusercontent.com/61790587/203078929-3cc6ad18-2685-42f4-96a2-dc8435611bee.png">
<img width="654" alt="Screenshot 2022-11-21 at 15 24 12" src="https://user-images.githubusercontent.com/61790587/203078955-cabaa7e6-6a7e-48e1-a648-8a79c6cd923a.png">

Reply:
- Underline removed.
- Hover and active has been activated on popup-button
- Popup window button resembles that of figma design